prow/Makefile

```diff
@@ -54,10 +54,7 @@ get-build-cluster-credentials:
 build:
```

Comment:
I would get rid of this as well, it should not be used either

Reply:
Is there some guidance I can write in the commit message about an alternate non-deprecated equivalent? Unlike for the `test` target, I don't see a `build` target in the top level Makefile.

Reply:
For now I just added a note about what this rule used to do, and how it's no longer encouraged.

Reply:
good point, build target is missing from top level, which should be added.

Reply:
Should the top level build target do what the old one used to do? Or something else?

Reply:
I would first ask who would use it.

Reply:
> build target is missing from top level, which should be added
> I would first ask who would use it.

These statements sound contradictory to me. Can you clarify? Namely:

(1) Do we want a `build` target in the top-level Makefile? And (2) if so, should it be the same rule as before (locally installing all binaries under `prow/cmd/...`)?